ndk/src/looper.rs

```diff
@@ -31,6 +31,7 @@ pub struct ThreadLooper {
 
 bitflags! {
     /// Flags for file descriptor events that a looper can monitor.
+    #[derive(Clone, Copy, Debug, Eq, Hash, Ord, PartialEq, PartialOrd)]
```

Comment:
Let's keep these in a somewhat consistent order (yes it's messy like this in `native_window.rs`/`surface_texture.rs` too):

```suggestion
    #[derive(Clone, Copy, Debug, Hash, PartialEq, Eq, PartialOrd, Ord)]
```